Project-Uni/RootShare
# Delete Communities

### Branched off of 271. No need to check until it's merged into master
Functionality to delete communities as an admin with FULL propagation

**Testing**
 - Create a new community and following communities, get followed by communities, leave some requests pending
 - Post as community, post in community, post externally
 - add profile and banner image
 - join the community with different account

**Deletion of community should clean all dependent arrays in other communities, users, and clean up posts, community_edges, and images**
